rofeeah_ayeni/alpha_challenges/conditionals.md

```diff
@@ -8,14 +8,30 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 ```ruby
 color = "Magenta"
 
-# write your code here
+if color == "Chartreuse"
```

Comment:
another way to skin a cat (uh, what weirdo came up with that saying?) is to use a `case` statement.

so instead of having to constantly say
`if color == "Chartreuse"`
`puts "blah"`
`elsif color  == "Magenta"`

you could have
````
case color
when "Magenta"
 puts "That's my favorite color"
when "Periwinkle"
 puts "This is my third favorite color"
```
and etc.

What you did was still correct, but this just looks a little nicer. 
Here's an [article with an example](https://www.rubyguides.com/2015/10/ruby-case/). Try it out for yourself!